trading_ig/rest.py

```diff
@@ -799,6 +799,9 @@ def update_working_order(
         session=None,
     ):
         """Updates an OTC working order"""
+        VERSION = 2
```

Comment:
This is just copied from create_working_order(): https://github.com/ig-python/ig-markets-api-python-library/pull/115/files#diff-a0fafc37c1b18ac26c2d6f35f497f1e1R735